libs/game/sprite.ts

```diff
@@ -142,6 +149,7 @@ class Sprite {
             case SpriteWriteProperty.AX: this.ax += value; break;
             case SpriteWriteProperty.AY: this.ay += value; break;
             case SpriteWriteProperty.Type: this.type += value; break;
+            case SpriteWriteProperty.Life: this.life += value; break;
```

Comment:
In general, wouldn't it be better to implement changeBy(p,v) { set(get(p)+v) } ?

Reply:
I could try but you can "get" more things than you can "set"

Reply:
the set will be a no-op then, just like now